### PR TITLE
KH2: Update the docs to support steam in the setup guide

### PR DESCRIPTION
**Pull request by JaredWeakStrike**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"

## What is this fixing or adding?
adds steam to the setup guide since its playable
added another faq at the bottom since its been a frequent asked question in the kh2 channel
## How was this tested?
I read it

### CODE REVIEW

File: worlds/kh2/docs/setup_en.md [hunk 1]
```diff
@@ -83,11 +83,13 @@ Have any questions on what's in logic? This spreadsheet made by Bulcon has the a
...
     - You need to trigger a cutscene or visit The World That Never Was for it to register that you have received the item.
 - What versions of Kingdom Hearts 2 are supported?
-    - Currently `only` the most up to date version on the Epic Game Store is supported: version `1.0.0.8_WW`.
+    - Currently the `only` supported versions are `Epic Games Version 1.0.0.9_WW` and `Steam Version Build Version 14716933` .
 - Why am I getting wallpapered while going into a world for the first time?
   - Your `Lua Backend` was not configured correctly. Look over the step in the [KH2Rando.com](https://tommadness.github.io/KH2Randomizer/setup/Panacea-ModLoader/) guide.
 - Why am I not getting magic?
     - If you obtain magic, you will need to pause your game to have it show up in your inventory, then enter a new room for it to become properly usable.
+- Why did I crash after picking my dream weapon?
+  - This is normally caused by having an outdated GOA mod or having an outdated panacea and or luabackend. To fix this rerun the setup wizard and reinstall luabackend and panacea. Also make sure all your mods are up-to-date.
```
Comment (Exempt-Medic) [inline worlds/kh2/docs/setup_en.md:92]:
```suggestion
  - This is normally caused by having an outdated GOA mod or having an outdated panacea and/or luabackend. To fix this rerun the setup wizard and reinstall luabackend and panacea. Also make sure all your mods are up-to-date. 
```

**Comment by Exempt-Medic** [comment]
I guess(?) this is a GitHub issue, but this file does not format/display properly on GitHub, though that's the case before and after these changes. The changes seem fine to me, I might try to investigate the error

**Review by Exempt-Medic — APPROVED** [review]

**Comment by Exempt-Medic** [comment]
I fixed the issues and opened a PR ( https://github.com/JaredWeakStrike/Archipelago/pull/5 ) addressing them.

**Review by Exempt-Medic — APPROVED** [review]
Reaffirming my approval

**Comment by rocket0634** [comment]
The KH2 Rando guide points to the latest release of OpenKH, as opposed to a specific version
Since the AP guide just says to follow the KH2 Rando guide to install OpenKH 3.2.0, it might be reasonable to omit the mod manager version from the AP guide.

If you plan on updating it manually any time it updates, the current AP guide should be corrected to OpenKH 3.3.0.

**Review by gaithern — APPROVED** [review]
Ran WebHost.py locally and looked over the changes to confirm what is new on the set up guide.  Information is clear and concise and the page looks great to me stylistically.

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**